tests/polars/test_polars_dataframe_generic.py

```diff
@@ -59,11 +59,11 @@ class Output(Input):
 
     @pa.check_types
     def fn(lf: LazyFrame[Input]) -> LazyFrame[Output]:
-        return lf.with_columns(col3=pl.lit(3.0))
+        return lf.with_columns(col3=pl.lit(3.0))  # type: ignore
```

Comment:
the assumption here is that if you're using @pa.check_types you're already coping without type annotations, no reason to put a bunch of casts in the tests. Maybe I'll have a look seperately and see if the decorator can be typed such that the signature is preserved